Source: Debian
Description: Use a flag, not an ioctl value, for HAVE_SENDBUF_IOCTL
 The preprocessor throws a hissy fit if we try to check for
 the truthiness of an ioctl definition itself, at least on PowerPC.
Author: Peter Pentchev <roam@ringlet.net>
Forwarded: not-yet
Last-Update: 2016-04-11

--- a/libscompat.m4
+++ b/libscompat.m4
@@ -377,7 +377,7 @@
 
 	return 0;
 }], [AC_MSG_RESULT(yes)
-     AC_DEFINE(HAVE_SENDBUF_IOCTL, TIOCOUTQ, [send buffer data])
+     AC_DEFINE(HAVE_SENDBUF_IOCTL, 1, [send buffer data])
      AC_DEFINE(SENDBUF_IOCTLVAL, TIOCOUTQ, [send buffer ioctl])],
     [AC_MSG_RESULT(no)])],
     [dnl assume no when cross-compiling
